sqlg-core/src/main/java/org/umlg/sqlg/structure/SqlgEdge.java

```diff
@@ -35,6 +35,8 @@ public SqlgEdge(SqlgGraph sqlgGraph, Long id, String schema, String table, SqlgV
         super(sqlgGraph, id, schema, table);
         this.inVertex = inVertex;
         this.outVertex = outVertex;
+        properties.clear();
```

Comment:
I followed the same logic that is present in https://github.com/pietermartin/sqlg/blob/master/sqlg-core/src/main/java/org/umlg/sqlg/structure/SqlgVertex.java#L811.

Note that this constructor is only called from https://github.com/pietermartin/sqlg/blob/master/sqlg-core/src/main/java/org/umlg/sqlg/structure/SqlgVertex.java#L375 and https://github.com/pietermartin/sqlg/blob/master/sqlg-core/src/main/java/org/umlg/sqlg/structure/SqlgVertex.java#L385, so `insertEdge` doesn't come into play here - we're constructing a new edge that is already present in the database from data that has already been loaded elsewhere.